RoadkillVocaDbAuth/VocaDbUserManager.cs

```diff
@@ -143,16 +147,20 @@ public override bool DeleteUser(string email)
 
 		public override string GetLoggedInUserName(HttpContextBase context)
 		{
-			if (FormsAuthentication.IsEnabled)
+			var cookie = context.Request.Cookies[".AspNetCore.Cookies"];
+			if (cookie != null)
 			{
-				if (context.Request.Cookies[FormsAuthentication.FormsCookieName] != null)
+				var cookieContainer = new CookieContainer();
+				cookieContainer.Add(new Cookie(name: cookie.Name, value: cookie.Value, path: cookie.Path, domain: ".vocadb.net"/* TODO: do not hardcode */));
+				using (var handler = new HttpClientHandler { CookieContainer = cookieContainer })
+				using (var client = new HttpClient(handler))

```

Comment:
Actually the HttpClient should not be created every time, but I'm not sure if it's really an issue, https://www.aspnetmonsters.com/2016/08/2016-08-27-httpclientwrong/

I'm sure there's a way to pass the cookie in some other way?

Reply:
I tried `SendAsync` along with `HttpRequestMessage.Headers.Add` instead of `GetStringAsync (this worked fine in a console application), but it causes a runtime error for some reason. I'll keep this as is for now.